examples/helloworld/src/index.js

```diff
@@ -1,4 +1,26 @@
 import React from 'react';
+import _ from 'underscore';
 import App from './App';
 
-React.render(<App />, document.getElementById('root'))
+
+var state = {
+    very: {
+        deeply: {
+            nested: {
+                counts: _.range(400).map(function () { return 0; })
+            }
+        }
+    }
+};
+
+function swapper(f) {
```

Comment:
Is there ever a good reason to want to tweak the swapper fn or the queueRender fn? E.g. to implement requestAnimationFrame batching?

We expected React.render to batch, but I'm not sure it's working properly. I maintain an application that has a lot of state churn during initialization (anti-pattern) and it causes a bunch of unwanted rendering. I haven't tried to figure it out yet.